google-cloud-bigtable/src/test/resources/logging-verbose.properties

```diff
@@ -0,0 +1,17 @@
+handlers= java.util.logging.ConsoleHandler
+.level= FINEST
+
+java.util.logging.ConsoleHandler.level = FINEST
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+io.grpc.level=FINEST
+# Example to customize the SimpleFormatter output format
+# to print one-line log message like this:
+#     <level>: <log message> [<date/time>]
+#
+
+#java.util.logging.SimpleFormatter.format=%4$s: %5$s [%1$tc]%n
+# time [level] loggerName: message
+java.util.logging.SimpleFormatter.format=%1$tT [%4$-7s] %2$s: %5$s%n
+
+# hide "Connecting to the Bigtable emulator at localhost:XXXX" lines
+com.google.cloud.bigtable.data.v2.BigtableDataSettings.level=WARNING
```

Comment:
please keep the level settings grouped (instead of splitting them around the formatting options)
(if this is copied from the existing logging config, please update that as well)